apps/nearbyflight/nearby_flight.star

```diff
@@ -0,0 +1,196 @@
+                                                    expanded = True,
+                                                    main_align = "space_evenly",
+                                                    children = [
+                                                        render.Text("%s" % nearest["status"], color = "#F09", font = "tom-thumb"),
+                                                    ],
+                                                ),
+                                            ],
+                                        ),
+                                    ],
+                                ),
+                                render.Row(
+                                    expanded = False,
+                                    main_align = "start",
+                                    cross_align = "end",
+                                    children = [
+                                        render.Column(
+                                            expanded = True,
+                                            main_align = "start",
+                                            cross_align = "center",
+                                            children = [
+                                                render.Padding(
+                                                    child = render.Text("%s" % nearest["flight"], color = "#0F0", font = "tom-thumb"),
+                                                    pad = (0, 3, 0, 0),
+                                                ),
+                                                render.Text("", color = "#0F0", font = "tom-thumb"),
+                                                render.Text("OPEN", color = "#666", font = "tom-thumb"),
+                                                render.Text("FLIGHT", color = "#666", font = "tom-thumb"),
+                                            ],
+                                        ),
+                                        render.Column(
+                                            expanded = True,
+                                            main_align = "start",
+                                            cross_align = "center",
+                                            children = [
+                                                render.Padding(
+                                                    child = render.Image(src = qr_code_for_flight(nearest)),
+                                                    pad = (5, 1, 0, 0),
+                                                ),
+                                            ],
+                                        ),
+                                    ],
+                                ),
+                            ],
+                        ),
+                    ],
+                ),
+            ],
+        ),
+    )
+
+def qr_code_for_flight(flight):
+    url = "https://app.airframes.io/flights/%s" % int(flight["id"])
```

Comment:
Probably want to add some caching around this call too? Just to keep request rates down. Your call, since you're running the API. =)

Reply:
Cached over here, but yeah, I will consider!